packages/next-app/src/utils/getTokenInfo.ts

```diff
@@ -0,0 +1,27 @@
+import tokensInfo from "public/tokens.json";
```

Comment:
Any reason why this dict couldn't include exactly the info you needed to avoid the parsing here?

Reply:
This is just to make it easier for us to change this later. Because this is an already made dict from Balancer it was our quick approach to get the token info data of the most used tokens (this doesn't cover all tokens, of course). Our goal is to use a request according to the specific tokens the user has